azure-pipelines.yml

```diff
@@ -110,8 +122,8 @@ jobs:
     - script: sh -x ./clean_build.sh
       displayName: Clean Build
     - task: PublishBuildArtifacts@1
-      displayName: Publish Release DMG
-      condition: eq(variables['B_BUILD_TYPE'], 'Release')
+      displayName: Publish Release DMG if on Big Sur agent
```

Comment:
I think that we don't need to include the condition here. The task will either not show or be skipped in azure UI and whoever looks into the file will see the condition check.

Reply:
The condition has been shortened to only release from Big Sur. This is future proof for when we ship for M1 (although this won't be for some time yet), and means that we don't publish every artifact from each build agent, which is what would happen if we didn't have the conditional. This will be pushed in due course, once I've addressed the rest of your feedback.

Reply:
Addressed in a677ab21.